www/tests/test_app.py

```diff
@@ -106,16 +86,16 @@ def test_register_with_web_update(testapp):
                   content_type='application/json'
     )
 
-    heartbeat_manager = testapp.HeartBeatManager(id=web_id,
-                                                 payload=payload,
-                                                 user_name='Bob',
-                                                 user_pass="asdf")
-    heartbeat_manager.create()
-    result = heartbeat_manager.thread_func(run_once=True)
-    assert result.status_code == 200
-    assert len(responses.calls) == 5
-    assert responses.calls[0].request.url == testapp.ROVERCODE_WEB_LOGIN_URL + '/'
-    assert responses.calls[1].request.url == testapp.ROVERCODE_WEB_LOGIN_URL + '/'
-    assert responses.calls[2].request.url == testapp.ROVERCODE_WEB_REG_URL + '/'
-    assert responses.calls[3].request.url == testapp.ROVERCODE_WEB_REG_URL + '?name=' + testapp.rover_name
-    assert responses.calls[4].request.url == testapp.ROVERCODE_WEB_REG_URL + '/' + str(web_id)+'/'
+    heartbeat_manager = testapp.HeartBeatManager(client_id='xxxx',
+                                                 client_secret="asdf")
+    heartbeat_manager.thread_func(run_once=True)
+    # test the side-effects of read()
+    assert heartbeat_manager.web_id == web_id
+    assert len(testapp.binary_sensors) == 2
+    assert testapp.binary_sensors[0].pin == 'Pin-A'
+    assert testapp.binary_sensors[1].pin == 'Pin-B'
+    assert responses.calls[0].request.url == testapp.ROVERCODE_WEB_OAUTH2_URL + '/'
+    assert responses.calls[1].request.url == testapp.ROVERCODE_WEB_REG_URL + '?client_id=xxxx'
+    # test that the update happened
+    # assert len(responses.calls) == 4
```

Comment:
Do you want this commented out?

Reply:
Nope! Fixed!

There should be a linter rule against commented out lines that look like code.